src/scores/probability/crps_impl.py

```diff
@@ -820,7 +820,7 @@ def crps_for_ensemble(
         ValueError: when method is not one of "ecdf" or "fair".
 
     See also:
```

Comment:
side-note: optionally you can use the in-built see also directive (I realise it's not actually part of this PR though, I did notice some places in the docs using the directive, while others don't).
```
..seealso::

    :py:func:`...`
```